lib/services/bluetooth_service.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_bluetooth_serial/flutter_bluetooth_serial.dart';
+
+class Session {
```

Comment:
The name of this class should be BuetoothService, Session doesn't really discribe what the class does.